src/encoded/commands/deploy.py

```diff
@@ -640,6 +621,15 @@ def hostname(value):
             args.candidate = False
         elif args.candidate:
             args.role = 'candidate'
+    # Set apache conf based on role and/or argument
+    available_cfgs = ['no-region', 'encoded']
```

Comment:
I find this a little confusing.   Should it maybe be encoded-no-region?

Reply:
yea, then the default could be an empty string.  encoded-apache.cfg, encoded-noregion-apache.cfg, encoded-worker-apache.cfg.